modules/installation-aws-upload-custom-rhcos-ami.adoc

```diff
@@ -0,0 +1,141 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/installing-aws-government-region.adoc
+
+[id="installation-aws-upload-custom-rhcos-ami_{context}"]
+= Uploading a custom {op-system} AMI in AWS
+ 
+If you are deploying to a custom Amazon Web Services (AWS) region, you must
+upload a custom {op-system-first} AMI that belongs to that region.
+
+.Prerequisites
+
+* Configure an AWS account.
+* Create an Amazon S3 bucket with the required IAM
+link:https://docs.aws.amazon.com/vm-import/latest/userguide/vmie_prereqs.html#vmimport-role[service role].
+* Upload your {op-system} VMDK file to Amazon S3. The {op-system} VMDK file must
+be the highest version that is less than or equal to the {product-title} version
+you are installing.
+* Download the AWS CLI and install it on your computer. See
+link:https://docs.aws.amazon.com/cli/latest/userguide/install-bundle.html[Install the AWS CLI Using the Bundled Installer (Linux, macOS, or Unix)].
+
+.Procedure
+
+. Export your AWS profile as an environment variable:
++
+[source,terminal]
+----
+$ export AWS_PROFILE=<aws_profile> <1>
+----
+<1> The AWS profile name that holds your AWS credentials, like `govcloud`.
+
+. Export the region to associate with your custom AMI as an environment
+variable:
++
+[source,terminal]
+----
+$ export AWS_DEFAULT_REGION=<aws_region> <1>
+----
+<1> The AWS region, like `us-gov-east-1`.
+
+. Export the version of {op-system} you uploaded to Amazon S3 as an environment
+variable:
++
+[source,terminal]
+----
+$ export RHCOS_VERSION=<version> <1>
+----
+<1> The {op-system} VMDK version, like `4.6.0`.
+
+. Export the Amazon S3 bucket name as an environment variable:
++
+[source,terminal]
+----
+$ export VMIMPORT_BUCKET_NAME=<s3_bucket_name>
+----
+
+. Create the `containers.json` file and define the credentials for your
```

Comment:
I'm not sure if `credentials` is the correct term here. It may confuse readers thinking we are referring to AWS credentials.

Reply:
Good point; I'll choose a better term :+1: